src/dotnet/Chat.UI.Blazor/Components/PlaceSettings/PlaceSettingsEditTypeModalPage.razor

```diff
@@ -0,0 +1,221 @@
+@namespace ActualChat.Chat.UI.Blazor.Components
+@using ActualChat.Invite
+@inherits ComputedStateComponent<PlaceSettingsEditTypeModalPage.ComputedModel>
+@{
+    var m = State.Value;
+    if (ReferenceEquals(m, ComputedModel.Loading))
+        return;
+
+    if (m.Place == null || !m.Place.Rules.CanEditProperties()) {
+        Context.Close();
+        return;
+    }
+    if (_form == null)
+        return;
+
+    Context.SetTitle("Place Type");
+    Context.ModalDataBag.Set(nameof(PlaceSettingsOwnerModalPage.ComputedModel.Place.IsPublic), m.IsPublic);
+    var canEditProperties = m.Place.Rules.CanEditProperties();
+}
+
+<Form Class="h-full" @ref="@_formRef" Model="@_form">
+    @{
+        _submitButtonInfo.CanExecute = _formRef?.IsValid ?? false;
+    }
+    <FormBlock Class="with-input-radio first">
+        <InputRadioGroup Name="chat_type" DisplayName="Chat type" @bind-Value="_form.IsPublic">
+            <TileItem Click="@(() => OnPublicChatClick(true))">
+                <Icon>
+                    <FormRadio Id="@_form.IsPublicTrueFormId" IsDisabled="@(!canEditProperties)" Value="@true" />
+                </Icon>
+                <Content>
+                    <Label InputId="@_form.IsPublicTrueFormId" Text="Public"></Label>
+                </Content>
+                <Caption>
+                    <Label
+                        InputId="@_form.IsPublicTrueFormId"
+                        Text="All members of this Place will see it in Place's chat list and will be able to talk there">
```

Comment:
Text seems incorrect. Currently we use 'Anyone can join this place by navigating to its URL'

Reply:
Done